axon-framework/events/event-processors/README.md

```diff
@@ -307,12 +307,12 @@ A `JpaSequencedDeadLetterQueue` configuration example:
 ```java
 public class DeadLetterQueueExampleConfig {
     
-    public ConfigurerModule configureDeadLetterQueueFor(String processingGroup) {
+    public ConfigurerModule configureDeadLetterQueueFor(EventProcessingConfigurer configurer) {
         return configurer -> configurer.eventProcessing(
             eventProcessingConfigurer -> eventProcessingConfigurer.registerDeadLetterQueue(
-                processingGroup,
+                "MyProcessingGroup",
```

Comment:
I'm wondering how many people will copy-paste `MyProcessingGroup` into their code and complain it doesn't work...

Perhaps we can add a notice here stating users should supply their actual processing group names?
Maybe a comment would do inside the code block, or a single sentence before the code-block starts.